projects/Mallard/src/helpers/files.ts

```diff
@@ -55,12 +55,11 @@ export const downloadNamedIssueArchive = (
     assetPath: string,
     filename: string,
 ) => {
-    const zipUrl = defaultSettings.zipUrl
     const returnable = RNFetchBlob.config({
         fileCache: true,
         overwrite: true,
         IOSBackgroundTask: true,
-    }).fetch('GET', `${zipUrl}/${assetPath}`)
+    }).fetch('GET', `${assetPath}`)
```

Comment:
```suggestion
    }).fetch('GET', `${apiUrl}${assetPath}`)
```

Reply:
Obviously we need to get apiUrl from somewhere ...

Reply:
Thanks @RichieAHB! I got it from the settings as best as I saw fit. A lot of things get it directly from `defaultSettings` but that will only ever be PROD AFAICS?